scripts/globals/spells/black/drain.lua

```diff
@@ -41,7 +41,7 @@ spellObject.onSpellCast = function(caster, target, spell)
         dmg = 0
     end
 
-    if target:isUndead() or target:hasImmunity(xi.immunity.DRAIN) then
+    if target:isUndead() then -- or target:hasImmunity(xi.immunity.DRAIN)
```

Comment:
Is there a reason to completely remove this? If what I put in before wasn't operating correctly, as I mentioned for the aspir on, can we fix the ENUM to work as it should? Maybe the xi.immunity list should be using the decimal numbers instead of hexadecimal is my first thought.

Reply:
definitely! I actually got caught up in an "emergency" of sorts and forgot about doing this.

Reply:
The problem I'm finding is that there is no immunity to Drain in the ENUM list. We might just have to make specific cases for mobs like jormy that aren't undead but are still immune by checking for mob pool ID?

Reply:
Ah then it was my bad completely to begin with. Disregard me.

Reply:
moreover I went ahead and checked for poolID for drain immunity since, unfortunately, we don't have an enum for drain immunity.